components/brave_shields/resources/panel/container.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+
+import MainPanel from './components/main-panel'
+
+function Container () {
+  return (
+    <MainPanel />
+  )
+}
+
+export default Container
```

Comment:
What's the point of this file? Why not just have the consumer render `<MainPanel />` directly?

Reply:
this file is very similar to the one in vpn. It will be used to render various view types.